scripts/cibuild.sh

```diff
@@ -54,10 +54,18 @@ if [[ ! -z $SYNTAX ]]; then
 fi
 
 echo -e "\n>> Running unit tests..."
-invoke unittest --failfast
+invoke unittest --failfast --buffer --keepdb
```

Comment:
Should this explicitly unset `NAUTOBOT_INTEGRATION_TEST`?

Reply:
I obviated the need for this environment variable. The default test runner doesn't pick up the tests in integration.

Reply:
Interesting, why doesn't it?

Reply:
Welp, I was wrong about that. Figured it out by toggling verbosity:

```
test_login (nautobot.core.tests.integration.test_home.HomeTestCase) ... ok
```